### ISSUE TITLE
Cannot run queue worker when maxworkers == 1

### ISSUE THREAD

**Issue by utdrmac** [dereuromark/cakephp-queue#404]
https://github.com/dereuromark/cakephp-queue/blob/master/src/Model/Table/QueueProcessesTable.php#L124

If max workers is 1, I get the following error:

```
$ bin/cake queue run -t ExportNotionDoc -v
Entity save failure. Found the following errors (server.validateCount: "Too many workers running (1/1). Check your `Queue.maxworkers` config.").
```

1 out of 1 worker is not too many. Shouldn't L124 above be `>` and not `>=`?

**Comment by dereuromark** [comment]
No, it means you are trying to add a 2nd one, but only allow one.
Set it to 2 then.

**Comment by dereuromark** [comment]
You could also still have one process Stuck, so maybe clear those out then.

**Comment by utdrmac** [comment]
I didn't have any other's running. That was the output the very first time I executed the `queue run` command. If that was the very first time I ran it, how could there be others in stuck state? I did change it to 2 to get it working, but it seems I shouldn't have to if I only want 1 max worker.

**Comment by dereuromark** [comment]
$currentWorkers = $this->find()->where(['server' => $value])->count();

So there is a worker already running then for some reason?
Maybe it is the own one?

**Comment by michielkeijts** [comment]
Have you checked your QueueProcesses Table? This should be empty. I guess there is still a row without a process attached to it?

**Comment by utdrmac** [comment]
This was too long ago. I simply increased maxworkers to 2 and the problem went away.
